### DISCUSSION TITLE
Request to join team

### DISCUSSION THREAD

**Discussion by Saikrishna1504** [ROM-builders/temporary#22461] [General]
Hey buddy ,
I am here again ,my commit history also good but rejected last time.once check please,since it is asking for need approval from many days it's hard for me to build official roms.

Successful Builds:-
Voltage os (official) : https://cirrus-ci.com/build/5361911974330368
Alpha Droid (unofficial) : https://cirrus-ci.com/build/5365518203355136

My Device trees : https://github.com/Saikrishna1504/android_device_redmi_begonia
And our organisation : https://github.com/Trinity-ROMS

Telegram I'd : https://t.me/Saikrishna1504

Hope I get this time

**Comment by hipexscape** [comment]
Wow

**Comment by Apon77** [comment]
Looks like you are inactive and also lack proof of #learn instruction. Please follow the #learn instructions. Sorry for the inconvenience.
